typescript/cli/src/commands/warp.ts

```diff
@@ -39,3 +47,36 @@ export const config: CommandModuleWithContext<{
     process.exit(0);
   },
 };
+
+export const read: CommandModuleWithContext<{
+  chain: string;
+  address: string;
+  out: string;
+}> = {
+  command: 'read',
+  describe: 'Reads the warp route config at the given path.',
+  builder: {
+    chain: {
+      ...chainCommandOption,
+      demandOption: true,
+    },
+    address: addressCommandOption(
+      'Address of the router contract to read.',
+      true,
+    ),
+    out: outputFileCommandOption('./configs/warp-route-config.yaml'),
+  },
+  handler: async ({ context, chain, address, out }) => {
+    const { multiProvider } = context;
+    const evmERC20WarpRouteReader = new EvmERC20WarpRouteReader(
+      multiProvider,
+      chain,
+    );
+    const warpRouteConfig = await evmERC20WarpRouteReader.deriveWarpRouteConfig(
+      address,
+    );
+    writeFileAtPath(out, warpRouteConfig + '\n');
```

Comment:
would be nice to log to stdout too and/or have a flag for this

Reply:
made `out` optional– so if present it writes, if not it's printed :)